test/audits/html/theme-colors.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const Audit = require('../../../audits/html/theme-color.js');
+const assert = require('assert');
+
+/* global describe, it*/
+
+describe('HTML: theme-color audit', () => {
+  const getMockHtml = (headInsert, bodyInsert) => {
+    headInsert = headInsert || '';
+    bodyInsert = bodyInsert || '';
+
+    const htmlString = `<!doctype html>
+      <html>
+        <head>
+          ${headInsert}
+          <title>Sample page</title>
+        </head>
+        <body>
+          ${bodyInsert}
+        </body>
+      </html>`;
+
+    const jsdom = require('jsdom').jsdom;
```

Comment:
will make that change.

Originally did this to try and please browserify but the ignore in gulp-browserify works regardless.